addons/popochiu/engine/popochiu.gd

```diff
@@ -27,12 +27,15 @@ var height := 0.0 : get = get_height
 var half_width := 0.0 : get = get_half_width
 var half_height := 0.0 : get = get_half_height
 var settings := PopochiuResources.get_settings()
+var projectSettings: RefCounted = null
```

Comment:
This variable seems to have very different formatting (camelCase instead of snake_case) compared to the rest of the script